src/components/Survey/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export const AnythingWrongSurvey = () => {
+  const { t } = useTranslation();
+  return (
+    <div className="margin-top-4">
+      <p>
+        <a
+          href="https://www.surveymonkey.com/r/DM6NYRX"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="Open EASi survey in a new tab"
+        >
+          {t('general:feedback.anythingWrong')}
+        </a>
+      </p>
+    </div>
+  );
+};
+
+/* There are multiple versions of the Improve EASi survey depending on format of
+  page it is being added to */
+
+// This is the common version for most pages
+export const ImproveEasiSurvey = () => {
+  const { t } = useTranslation();
+  // Period and space to be added between translations since they are on same line
+  const periodSpace = '. ';
+  return (
+    <div className="margin-top-4">
+      <p>
+        {t('general:feedback.improvement') + periodSpace}
```

Comment:
```suggestion
        {`${t('general:feedback.improvement')}. `}
```

Alternatively, you can also probably use `Trans` in this situation too.

Reply:
This was a Trans when I did it originally but b/c of the need to support the other header style survey, and there was no easy way of doing that with a Trans block